rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -123,7 +123,10 @@ public G.CompilationUnit visit(SourceUnit unit, ModuleNode ast) throws GroovyPar
         }
 
         for (ClassNode aClass : ast.getClasses()) {
-            if (aClass.getSuperClass() == null || !"groovy.lang.Script".equals(aClass.getSuperClass().getName())) {
+            if (aClass.getSuperClass() == null
+                    || !("groovy.lang.Script".equals(aClass.getSuperClass().getName())
+                            || "RewriteGradleProject".equals(aClass.getSuperClass().getName())
+                            || "RewriteSettings".equals(aClass.getSuperClass().getName()))) {
```

Comment:
The super class of the Gradle scripts are now explicitly `RewriteGradleProject` or `RewriteSettings`, so we need to exclude these super types from being transformed into the rewrite LST.